### PR TITLE
PPC 2.0: thumbnails in checklist ignore page background

### LINKED ISSUE #8271: PPC - thumbnails in checklist ignore page background
## Bug Description

Originally reported by @choumx 
page thumbnails in checklist appear to ignore background images (non-background images are OK)

Looks like it might be a container issue in the checklist because the background element is showing up, but it looks like it’s getting pushed off to the side (see below screenshot)

## Expected Behaviour

Page thumbnails should closely resemble their associated pages 

## Additional Context

- Plugin Version:
- WordPress Version:
- Operating System:
- Browser:

---

_Do not alter or remove anything below. The following sections will be managed by moderators only._

### PR DESCRIPTION
**Pull request by samwhale**
## Context

PPC 2.0 bug found in bugbash

## Summary

The `Background` component in the `Thumbnail` was styling the `PagePreview` component in ways that we did not want.

## Relevant Technical Choices

The `Thumbnail` component had some styles that targeted all `img` and `video` tags. This was nice in Storybook when we weren't using the `PagePreview` component.

However, the `PagePreview` component (along with the `UnitProvider`) does its own calculations into the spacing and display of all `img` and `video` elements.

I removed the styles that targeted all of the `img` and `video` tags. Now those styles will be the responsibility of the child rather than the `Background` component.

## To-do

n/a

## User-facing changes

|Before|After|
|--|--|
|not-it|it|

## Testing Instructions

1. Open editor
2. Add an image to the page background. If you want, add more things to your story.
3. Add pages until the checklist has the `Design` section open.
4. The story preview should look like a tiny version of your story 🎉

### QA

- [ ] This is a non-user-facing change and requires no QA

### Does this PR have a security-related impact?

no

### Does this PR change what data or activity we track or use?

no

### Does this PR have a legal-related impact?

no

## Checklist

- [x] This PR addresses an existing issue and I have linked this PR to it in ZenHub
- [x] I have tested this code to the best of my abilities
- [x] I have verified accessibility to the best of my abilities ([docs](https://github.com/google/web-stories-wp/blob/main/docs/accessibility-testing.md))
- [x] I have verified i18n and l10n (translation, right-to-left layout) to the best of my abilities
- [ ] This PR contains automated tests (unit, integration, and/or e2e) to verify the code works as intended ([docs](https://github.com/google/web-stories-wp/tree/main/docs#testing))
- [x] I have added documentation where necessary
- [x] I have added a matching `Type: XYZ` label to the PR

---

Fixes #8271

### CODE REVIEW

File: assets/src/edit-story/components/thumbnail/styles.js [hunk 1]
```diff
@@ -107,14 +107,6 @@ export const Background = styled.div(
     display: flex;
     border-radius: ${theme.borders.radius.small};
 
-    img,
```
Comment (BrittanyIRL) [inline assets/src/edit-story/components/thumbnail/styles.js:110]: 😿 
Screen Shot 2021-07-15 at 11 08 56 AM

Comment (maxyinger) [reply assets/src/edit-story/components/thumbnail/styles.js:110]: maybe let's add some more specific selector here that applies these styles to images/videos rendered, but doesn't reach in and effect images/videos within a rendered story

Comment (samwhale) [reply assets/src/edit-story/components/thumbnail/styles.js:110]: ah good catch - I'll fix that :)

**Review by maxyinger — APPROVED** [review]
Just gotta fix the scenario where it's a thumbnail with an img/video directly rendered instead of a story preview, but other than that looks gucci

File: assets/src/edit-story/components/thumbnail/styles.js (hunk 1 above)
Comment (samwhale) [reply assets/src/edit-story/components/thumbnail/styles.js:110]: Should be fixed now!

File: assets/src/edit-story/components/thumbnail/styles.js [hunk 2]
```diff
@@ -107,8 +107,12 @@ export const Background = styled.div(
...
-    video {
+    /*
+    Only reach one level deep so that images and videos
+    in stories are not affected.
+    */
+    & > img,
```
Comment (BrittanyIRL) [inline assets/src/edit-story/components/thumbnail/styles.js:114]: 🎉 perfect, thank you for debugging this!

**Review by BrittanyIRL — APPROVED** [review]

**Pull request merged**